src/one_dragon_qt/app/directory_picker.py

```
@@ -205,12 +252,34 @@ def _on_browse_clicked(self):
+        self.progress_bar.start()
+
+        self._runner = UnpackResourceRunner(self.installer_dir, self.selected_path)
+        self._runner.log_message.connect(self._on_unpack_log)
+        self._runner.finished.connect(self._on_unpack_finished)
+        self._runner.start()
+
+    def _on_unpack_log(self, message: str) -> None:
+        """更新单行状态标签"""
+        self.status_label.setText(message)
+
+    def _on_unpack_finished(self, success: bool) -> None:
```

Ruff (0.15.1)

[warning] Unused method argument: `success`

(ARG002)